language/evm/move-to-yul/src/functions.rs

```diff
@@ -901,7 +943,7 @@ impl<'a> FunctionGenerator<'a> {
             // and if it is set, abort. Otherwise set this bit.
             let exists_call = self.parent.call_builtin_str(
                 ctx,
-                YulFunction::StorageLoadU8,
+                YulFunction::AlignedStorageLoad,
```

Comment:
We need to use this instead of LoadU8 because the RESOURCE_EXISTS_FLAG_SIZE is 32 bytes?

Reply:
Yes, its 32 bytes to avoid that all subsequent fields are not on word boundary. Then loading a u8 from a 32 byte location is actually more expensive than just the plain full word ;-)